x-pack/solutions/security/plugins/security_solution/public/onboarding/components/onboarding_body/cards/siem_migrations/start_migration/start_migration_check_complete.ts

```diff
@@ -19,12 +19,11 @@ export const checkStartMigrationCardComplete: OnboardingCardCheckComplete<
 
   let isComplete = false;
 
-  if (missingCapabilities.length === 0) {
+  if (siemMigrations.rules.isAvailable() && missingCapabilities.length === 0) {
```

Comment:
NIT: The capabilities check is already done internally by the `isAvailable`, there's no need to check it again here.
```suggestion
  if (siemMigrations.rules.isAvailable()) {
```

Reply:
Oh right you told me exactly that. 🤦 .